src/mbgl/util/mat3.cpp

```diff
@@ -94,4 +95,11 @@ void matrix::scale(mat3& out, const mat3& a, double x, double y) {
     out[8] = a[8];
 }
 
+void transformMat3f(vec3f& out, const vec3f& a, const mat3& m) {
```

Comment:
Our matrix functions don't have the type of the argument in their name. C++ can automatically select the correct overload when passing a `vec3f`/`mat3`. However, given that we added the 4-component equivalents back in https://github.com/mapbox/mapbox-gl-native/commit/cef1b254445bdcfb20ada157ecc7f72c27aeacf9 with a similar name, this is probably okay.